appyters/dimensionality_reduction_visualization/dimensionality_reduction_visualization.ipynb

```diff
@@ -372,11 +353,14 @@
     "%%appyter code_exec\n",
     "{% if need_normalize.value%}\n",
     "# Normalize Data\n",
+    "def log2_normalize(x, offset=1.):\n",
+    "    return np.log2(x + offset)\n",
+    "\n",
     "df_data_norm = log2_normalize(expr_df, offset=1)\n",
     "df_data_norm = qnorm.quantile_normalize(df_data_norm, axis=0)\n",
     "\n",
     "#convert to zscores\n",
-    "expr_df = pd.DataFrame(zscore(df_data_norm, axis=1), index=df_data_norm.index, columns=df_data_norm.columns)\n",
+    "expr_df = pd.DataFrame(zscore(df_data_norm, axis=0), index=df_data_norm.index, columns=df_data_norm.columns)\n",
```

Comment:
I'm a little confused about the orientation. is the dataframe genes on rows/samples on columns or the other way around

Reply:
The dataframe has columns to be gene. In this case, do we set axis to be 0?
![Screen Shot 2022-12-01 at 1 14 23 PM](https://user-images.githubusercontent.com/53023425/205129252-38f2d403-7859-44d4-8174-259918074e7f.png)

Reply:
The axis is correct if this is the orientation. Though we should use the other orientation for display since the standard is the other way.

Reply:
Sure. I will display the data input before the transpose.